.golangci.yml

```diff
@@ -388,6 +388,11 @@ issues:
       linters:
         - forbidigo
 
+    - path: discover/apispecs_test.go
+      text: "Possible resource leak"
```

Comment:
This was caused by a UT, where I create and defer a GenServer() only needed for UTs.
The server instance is spun up and brought for each iteration of table tests.